### PR TITLE
GPC-78: add api versioning

### PR DESCRIPTION
**Pull request by CarlyG55**

### CODE REVIEW

File: .github/workflows/build-and-push.yml [hunk 1]
```diff
@@ -71,7 +86,7 @@ jobs:
     if: inputs.push
     strategy:
       matrix:
-        env: [dev, demo]
+        env: [ dev, demo ]
```
Comment (olevett) [inline .github/workflows/build-and-push.yml:89]: Your yaml linter is very agressive!

**Comment by olevett** [comment]
This seems fine, to check this is going to give us incrementing builds, one per commit since the 0.0.1 tag?

**Review by olevett — APPROVED** [review]

File: .github/workflows/build-and-push.yml (hunk 1 above)
Comment (CarlyG55) [reply .github/workflows/build-and-push.yml:89]: I don't even know where it's come from 😅

**Comment by CarlyG55** [comment]
> This seems fine, to check this is going to give us incrementing builds, one per commit since the 0.0.1 tag? I think I'd be keen to put _some_ information in the tag, like `version-0.0.1`, and tell GitVersion what the prefix is, but I'm not too worried by this.

From what I can tell, yes - I'll give this a go

**Pull request merged**